tests/sdk-consumer/app.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use \SwedbankPay\Api\Client;
+
+$client = new \SwedbankPay\Api\Client\Client();
+$version = new \SwedbankPay\Api\Client\Version();
+
+$v = $version->getVersion();
+echo "version: $v";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '"version: $v"'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)